appveyor.yml

```diff
@@ -185,6 +185,9 @@
 
             if ($env:BOND_BUILD -eq "C++") {
 
+                $env:PreferredToolArchitecture = "x64"
```

Comment:
We can leave this above and have it apply to everything. It shouldn't hurt and might help.